doc/contribute/index.rst

```diff
@@ -130,6 +130,22 @@ to add the sign-off you can also amend a previous commit with the sign-off by
 running ``git commit --amend -s``. If you've pushed your changes to GitHub
 already you'll need to force push your branch after this with ``git push -f``.
 
+Notes
+=====
+
+Any contributions made as part of submitted pull requests are considered free
```

Comment:
Trying to capture @galak 's input here: adding: "Any contributions made as part of submitted pull requests are considered free for the Project to use."

Reply:
do we need to differentiate between DNM PRs where maybe the developer is asking for feedback?

Reply:
@dleach02 I don't think so; and this was not brought up in the meetings as far as I remember. (Also not everyone can add a label to a PR such as DNM, RFC, only collaborators can do that.) If you see a reason to differentiate, pls elaborate.

Reply:
Not DNM, but maybe draft (IIRC, everyone can do this)?

Reply:
I think a case could be made that if it's a draft PR it's not a submitted pull request, so no it's not free for the project to use.  But since "ready" PRs can be converted back to "draft" there's a problem where permission could be revoked after a commit had been cherry-picked into another PR.

The only case I think is clear is that if a commit doesn't have an SOB line it can't be adopted, regardless of what sort of PR it's in.

Reply:
@pabigot is right on PR status - they can switch back and forth, so this won't work.

Reply:
> The only case I think is clear is that if a commit doesn't have an SOB line it can't be adopted, regardless of what sort of PR it's in.

Yea, and I believe this is already covered. When cherry-picking we need to preserve the SOB. If there's no SOB, we don't cherry pick.